first session/assignment/shoppingmall.html

```diff
@@ -0,0 +1,191 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>나만의 쇼핑몰</title>
+    <link rel="stylesheet" href="shoppingmall.css">
+</head>
+<body>
+    <header>   
+        <div class="header-icon">🛍️</div>
+        <h1>민준이의 쇼핑몰</h1>
+        <!-- <div class="menu-bar">☰</div> -->
+        <!--<img src="./static/img/menu-bar.png" alt="menu bar">-->
+        <div class="hamburger-menu">
+            <button class="hamburger-icon" onclick="toggleMenu()">
```

Comment:
요 toggleMenu는 어떤 녀석일까요~??

Reply:
추가적으로 의미론적인 관점에서 button에는 type=button을 지정하는 것이 좋습니다!!
```suggestion
            <button class="hamburger-icon"  type="button">
```
참고할만한 링크도 남겨둘게요!
https://nykim.work/96